testing/README.md

```diff
@@ -0,0 +1,35 @@
+# Helpful settings for testing
+
+The plugin will run if you have configured kubectl locally (and have the permissions required to watch/list/read secrets).  
+For full integration testing then this directory contains some files useful for *developer* testing the plugin inside a Jenkins running inside a kubernetes (tested with GKE).
+
+They may require some small tweaks for your environment (as it will be different to mine), but if so please don't attempt to commit them back :-)
+
+## Testing
+
+### Initial setup...
+
+1. Create the testing namespace  `kubectl apply -f testing-namespace.yaml`

```

Comment:
FYI, if it's easier, you can concatenate all these files together so you only have to do one `kubectl apply -f`. Though it's more explicit what's happening if it's 4 separate files. Your choice